source/index.md

```diff
@@ -600,9 +602,46 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required. This must be
+derived according to the documentation.
```

Comment:
I would not mention the token and merchant ID, as these are evident from the list below. Two simpler sentences are e.g. "The shared secret must be derived from the ephemeral public key and your private key. The cryptography involved in Google Pay is described here.". If this suggestion is used, I suggest the text "The shared secret derived from the ephemeral public key and your private key, Base64 encoded." in the list below is changed to "The shared secret. Base64 encoded." to avoid repetition.